constellation-app/constellation
# Keep Attribute Editor multiple values expanded

### Description of the Change
When the graph selection was changing, or graph was zooming in or out , any expanded node or transaction attributes were contracting to display "Multiple Values", rather than remaining expanded and populating newly selected values. 



### Alternate Designs
A possible extension is to keep the selection of attributes across graphs, and/or retain the selection upon returning to the graph. Currently it works only within a graph and loses the selection if the user leaves and returns a graph.


### Why Should This Be In Core?
Better UX and saves time


### Benefits
Better UX and saves time


### Possible Drawbacks
N/A


### Verification Process

1. Open a graph and the Attribute Editor
2. Select a set of nodes or transactions such that an attribute has more than one value for the selection
3. Expand an attribute with multiple values
4. Change the graph selection such that the chosen attribute still has multiple values and verify the expanded attribute/s do not collapse. 
5. Select a single Node or Transaction and verify the attribute/s collapse. Select multiple Nodes or Transactions and verify the previously viewed attribute/s expand again.
6. Zoom in and out the graph and verify the attributes are still expanded. 


 
### Applicable Issues


https://github.com/constellation-app/constellation/issues/1128
